Apps/Src/ReadCarCAN.c

```diff
@@ -1,13 +1,20 @@
 /* Copyright (c) 2020 UT Longhorn Racing Solar */
 
+#define __TEST_READCARCAN
+// Uncomment the #define above when testing Renode or on hardware. Also uncomment #define in the h file
+
```

Comment:
If we're going to use this to test (which I already explained why we shouldn't, but if we need to on Saturday) don't define it in both places. By including ReadCarCAN.h, it will be defined in this file as well since it is defined there.
```suggestion
```